_rules/SC4-1-2-iframe-unique-name.md

```diff
@@ -0,0 +1,227 @@
+---
+name: iframe has a unique accessible name
+
+description: |
+ The accessible name for each iframe is unique
+
+success_criterion:
+- 4.1.2
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Jey Nandakumar
+---
+
+## Test procedure
+
+### Applicability
+
+The rule applies to `iframe` elements that are [exposed to assistive technologies](#exposed-to-assistive-technologies) and have a [non-empty](#non-empty) [accessible name](#accessible-name).
+
+### Expectation
+
+Each target element has an [accessible name](#accessible-name) that is unique within all applicable `iframes` within the same [document tree](https://www.w3.org/TR/dom41/#document-trees) or [shadow tree](https://www.w3.org/TR/dom41/#shadow-trees), where the `src` attributes are different.
+
+**Note:** 
+- An `iframe` creates an embedded browsing context. The content of this browsing context is not part of the [document tree](https://www.w3.org/TR/dom41/#document-trees) or [shadow tree](https://www.w3.org/TR/dom41/#shadow-trees) that it is embedded in.
```

Comment:
This note seems to be missing a conclusion. It's stating something, but it isn't relating it to the expectation.

Reply:
There is no conclusion for this note. It is purely a definition/ note on what an `iframe` is, and what is it not a part of.

Reply:
I'm thinking that Notes should be in place to describe specific caveats or considerations for the rules, not for defining terms used in the rule.

Reply:
Should there be a definition of iframe?